packages/patrol_finders/test/patrol_tester_test.dart

```diff
@@ -211,6 +292,87 @@ void main() {
           throwsAssertionError,
         );
       });
+
+      group('with alignment', () {
+        patrolWidgetTest('finds no widgets', (tester) async {
+          const width = 300.0;
+          setPhysicalSize(tester.tester, width);
+
+          await tester.pumpWidget(
+            MaterialApp(
+              home: Scaffold(
+                body: SingleChildScrollView(
+                  scrollDirection: Axis.horizontal,
+                  child: Container(
+                    color: Colors.blue,
+                    width: 2 * width,
+                    child: const Row(
+                      mainAxisAlignment: MainAxisAlignment.center,
+                      children: [
+                        SizedBox(
+                          width: width,
+                          child: TextField(),
+                        ),
+                      ],
+                    ),
+                  ),
+                ),
+              ),
+            ),
```

Comment:
Can we extract this widget into something more generic that will take `width` and `testedWidget` as a parameter from outside? We could then add some comments to this generic widget with explanation why we set those width in such way etc.